mylyn.tasks/org.eclipse.mylyn.tasks.ui.tests/src/org/eclipse/mylyn/internal/tasks/ui/migrator/ConnectorMigratorTest.java

```diff
@@ -424,7 +422,7 @@ public void migrateTasks() throws Exception {
 				manager.getRepositoryConnector("mock.new"), monitor);
 
 		verify(migrator, never()).getTaskData("key1", newConnector, newRepository, monitor);
-		verify(migrator, never()).createTask(argThat(not(taskData2)), any(TaskRepository.class));
+		// verify(migrator, never()).createTask(argThat(not(taskData2)), any(TaskRepository.class));
```

Comment:
Ed @merks please create an issue to enable this test back and mention it here, otherwise we will forget about it.

Reply:
Issue https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/146 created and mentioned.